vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -10,7 +10,7 @@ In addition, this document describes the series of `ent ecr` commands that manag
 
 ## Entando 7.1 Bundle Development
 
-Beginning with Entando 7.1, the `ent bundle` command and its convenience methods introduce a streamlined process to govern the structure, files and management of Entando bundles. The bundle development lifecycle consists of 6 stages, each corresponding to a subcommand:
+Beginning with Entando 7.1, the `ent bundle` module introduces a streamlined process to govern the files, structure, and management of Entando bundles. The bundle development lifecycle consists of 6 stages, each corresponding to a subcommand:
```

Comment:
`ent bundle cli` tool (per discussion)

structure is the broadest term in that list and encompasses files, so the list should lead with it

Reply:
My understanding is that ascending or descending is fine. In this case, because I wanted end with the management, I picked ascending. The thinking was that management is bigger than structure since it controls it.